packages/core/js-client/src/jsPeer/errors.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class ExpirationError extends Error {}
+
+export class InterpreterError extends Error {}
+
+export class SendError extends Error {}
```

Comment:
Could you please explain what do these solve exactly?

Reply:
I will add comments.
I separated errors to improve assertion with `instanceof` operator

Reply:
are these errors exposed to the user so he can use `instanceof`? Or you only use `instanceof` yourself somewhere?

Reply:
They used in tests right now. Helps to validate specific error. It's just replacement of the old bad API with 10 different statuses. I will add more errors to cover every case and open them for user. Right now if user wants to check error type, he can use error `message` field